--- linux-3.18.4/kernel/fork.c	2015-01-27 09:30:10.000000000 -0700
+++ Kernel/linux-3.18.4/kernel/fork.c	2015-02-01 15:46:17.655276881 -0700
@@ -281,7 +281,7 @@
 	if (max_threads < 20)
 		max_threads = 20;
 
-	init_task.signal->rlim[RLIMIT_NPROC].rlim_cur = max_threads/2;
+	init_task.signal->rlim[RLIMIT_NPROC].rlim_cur = max_threads*95/100;
 	init_task.signal->rlim[RLIMIT_NPROC].rlim_max = max_threads/2;
 	init_task.signal->rlim[RLIMIT_SIGPENDING] =
 		init_task.signal->rlim[RLIMIT_NPROC];
